electron/constants.ts

```diff
@@ -114,16 +114,24 @@ function getSteamCompatFolder() {
   return `${userHome}/.steam/steam`
 }
 
-export function getSteamLibraries() {
+export function getSteamLibraries(): string[] {
   const vdfFile = join(steamCompatFolder, 'steamapps', 'libraryfolders.vdf')
+  const libraries = ['/usr/share/steam']
```

Comment:
Under which conditions does this `/usr/share/steam` library get created? It doesn't exist for me

Reply:
I believe if you install proton-ge from AUR.